packages/firebase_ml/lib/src/firebase_remote_model.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A remote model to be downloaded to the device.
+///
+/// https://firebase.google.com/docs/reference/android/com/google/
+/// firebase/ml/common/modeldownload/FirebaseRemoteModel
+abstract class FirebaseRemoteModel {
+  /// Name associated with remote model in the Firebase console.
+  final String modelName;
+
+  /// Constructor for [FirebaseRemoteModel] that requires
```

Comment:
nit: The first sentence should be a summary on one line. [style guide](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph)